tests/bridge/integration/peg_in/peg_in.rs

```diff
@@ -0,0 +1,225 @@
+use std::time::Duration;
+
+use bitcoin::{Amount, OutPoint};
+
+use bitvm::bridge::{
+    connectors::{connector::P2wshConnector, connector_0::Connector0},
+    graph::{FEE_AMOUNT, INITIAL_AMOUNT},
+    scripts::generate_pay_to_pubkey_script_address,
+    transactions::{
+        base::{BaseTransaction, Input},
+        peg_in_confirm::PegInConfirmTransaction,
+        peg_in_deposit::PegInDepositTransaction,
+        peg_in_refund::PegInRefundTransaction,
+    },
+};
+use esplora_client::Error;
+use tokio::time::sleep;
+
+use crate::bridge::{helper::generate_stub_outpoint, setup::setup_test};
+
+#[tokio::test]
+async fn test_peg_in_success() {
+    let (client, depositor_context, _, verifier_context, _, _, _, _, _, _, _, _, _) = setup_test();
+
+    let evm_address = String::from("evm address");
```

Comment:
We have a placeholder EVM address in graph::EVM_ADDRESS already. Maybe better to use that for now so it's easier to change this in the future if needed.

Reply:
Done, now we use `evm_address` from `context`.